tools/wt_binary_decode.py

```diff
@@ -365,8 +365,8 @@ def block_decode(p, b, nbytes, opts):
     if disagg_delta:
         # WT_BLOCK_HEADER in block.h (44 bytes)
         blockhead = btree_format.BlockHeader.parse(b_page, disagg=opts.disagg)
-        # WT_DELTA_HEADER in btmem.h (20 bytes)
-        deltahead = btree_format.DeltaHeader.parse(b_page)
+        # WT_PAGE_HEADER in btmem.h (28 bytes)
+        deltahead = btree_format.PageHeader.parse(b_page)
```

Comment:
Do we need to distinguish deltahead and pagehead? Can we just have header? Can we extract some of the code outside the if block?

Reply:
I don't think we can extract the code, as I think the ordering of the page and block header does matter.